doc/source/releases/18.09_announce.rst

```diff
@@ -0,0 +1,87 @@
+
+===========================================================
+September 2018 Galaxy Release (v 18.09)
+===========================================================
+
+.. include:: _header.rst
+
+Highlights
+===========================================================
+
+**Extensive Workflow Enhancements**
+  Workflows got a lot of love this time around, with new runtime parameters for
+  subworkflows, exposed workflow versions, and zoom capability in the editor, to
+  name a few.  There were also a number of usability enhancements including
+  better labeling, links, overhauled workflow import interfaces, and many more.
+
+  `Pull Request 6664`_, `Pull Request 6683`_, `Pull Request 6142`_,
+  `Pull Request 6240`_, `Pull Request 6290`_, `Pull Request 6428`_,
+  `Pull Request 6441`_, `Pull Request 6580`_, `Pull Request 6596`_,
+  `Pull Request 6678`_, `Pull Request 6076`_, `Pull Request 6274`_,
+  `Pull Request 6570`_, `Pull Request 6680`_, `Pull Request 6774`_
+
+**Group Tags**
```

Comment:
@mvdbeek is there a training or tutorial or intro to group tags usage? Would be cool to include it.

Reply:
No, we need some tools first, I'm working on this.